### ISSUE TITLE
Availibility of coordinates should be check

### ISSUE THREAD

**Issue by kelson42** [openzim/mwoffliner#670]
Scraping with `1.8.2` seems, depending of the wiki, to generate a lot of following warnings:
```
The response contained warnings: { main: { '*': 'Unrecognized parameter: colimit.' },
  query:
   { '*': 'Unrecognized value for parameter "prop": coordinates.' } }
```

The reason is that it seems that Mediawiki is configurable to support - or don't support - API queries about geo-coordinates. If not, we get this warnings. This is the case for example with de.wikibooks.de. See #663.

We should, at start, detect if the remote Mediawiki support this or not, and if not, stop asking about the coordinates in our API requests. This should avoid the ''silly'' warnings.

**Comment by kelson42** [comment]
We still have a one warning here.
```
[info] [2019-07-04T12:42:21.454Z] Getting JSON from [http://species.wikimedia.org/w/api.php?action=query&format=json&prop=redirects%7Crevisions%7Cpageimages%7Ccoordinates&rdlimit=max&rdnamespace=0&titles=Main_Page&colimit=max]
[warn] [2019-07-04T12:42:21.693Z] Got warning from MW Query {
	"main": {
		"*": "Unrecognized parameter: colimit."
	},
	"query": {
		"*": "Unrecognized value for parameter \"prop\": coordinates."
	}
}
```

Like for other things, this should be properly tested (without error) and then the whole API requests should be done with the adequate parameters.

**Comment by ISNIT0** [comment]
This first request is used to check for the existence of coordinates.
The log is at a low level so we can always see when there is a warning

**Comment by kelson42** [comment]
Then it should be a normal log entry saying something like "Checking for availibility of coordinates: no"

**Comment by kelson42** [comment]
@ISNIT0 Actually here we face again a wrong design pattern which has been used and fixed already in a few other places: the fallback, instead of checking properly the capacity of the backend and act accordingly. This was the case for the "offline.css",or the Parsoid/MCS stuff I believe and here we have to fix it to if want to have a proper handling and stop misleading users with wrong log messages.

**Comment by kelson42** [comment]
This seems to not work, tested with 1.9.4:
```
[info] [2019-07-21T12:08:42.120Z] Getting JSON from [https://hitchwiki.org/en/api.php?action=query&format=json&prop=redirects%7Crevisions%7Cpageimages%7Ccoordinates&rdlimit=max&rdnamespace=0&titles=Main_Page&colimit=max]
[warn] [2019-07-21T12:08:42.245Z] Got warning from MW Query {
	"main": {
		"*": "Unrecognized parameter: colimit."
	},
	"query": {
		"*": "Unrecognized values for parameter \"prop\": pageimages, coordinates."
	}
}
```

**Comment by ISNIT0** [comment]
This was merged into master 10 days ago, but 1.9.4 was released 14 days ago. So I wouldn't expect it to be fixed in 1.9.4
